app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,23 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
+  end
+
+  def logged_in?
+    !current_user.nil?
```

Comment:
tìm hiểu về `present?` hì

và sử dụng thay vì nil? và phải nghịch đảo kết quả

Reply:
> tìm hiểu về `present?` hì
> 
> và sử dụng thay vì nil? và phải nghịch đảo kết quả

present? Dùng để check object có NOT nil, NOT false, hay là NOT empty, không có chuỗi rỗng hay không?